content/en-us/reference/engine/classes/Part.yaml

```diff
@@ -24,17 +24,20 @@ properties:
       The `Enum.PartType` enum controls the shape value, and has five possible
       shapes:
 
-      | Shape/Value | Description                             |
-      | ----------- | --------------------------------------- |
-      | Ball        | A spherical shape, like a basketball.   |
-      | Cylinder    | A rod-like shape, like a tin can.       |
-      | Block       | The default, brick shape.               |
-      | Wedge       | A wedge shape with a slope on one side. |
-      | CornerWedge | A wedge shape with slopes on two sides. |
+      | Shape/Value | Description                                 |
+      | ----------- | ------------------------------------------- |
+      | Ball        | A spherical shape.                          |
+      | Block       | A block shape.                              |
+      | Cylinder    | A cylinder shape oriented along the X axis. |
```

Comment:
```suggestion
      | Cylinder    | A cylinder shape.                           |
```

Reply:
Lets keep the X axis citation off... it uses that axis when you change a Part to "Cylinder" shape, but when you insert a Cylinder from Studio's toolbar, it uses the Y axis. In other words, the axis is not specifically related to the Shape property itself. :)